upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -451,27 +471,30 @@ spec:
       serviceAccountName: karpenter
       dnsPolicy: Default
       containers:
-        - name: manager
-          image: public.ecr.aws/karpenter/controller:v0.6.0
-          resources: 
+        - name: controller
```

Comment:
as of v0.6.2 of Karpenter, we're running the controller and webhook container in the same pod. Do you intentionally not want to do that in kOps?

Reply:
Hm. Usually webhooks are horisontally scalable, while controllers are leader elected. So typically they should be split. Any benefits to merging them?

Reply:
We may as well merge since both use the same serviceaccount as well. Webhook shouldn't really have access to the AWS API or anything like that. I doubt that is what is causing this issue though.